code/modules/atmospherics/environmental/LINDA_system.dm

```diff
@@ -73,6 +73,7 @@
 			if (current_turf.atmos_adjacent_turfs)
 				current_turf.atmos_adjacent_turfs -= src
 			UNSETEMPTY(current_turf.atmos_adjacent_turfs)
+		SEND_SIGNAL(current_turf, COMSIG_TURF_CALCULATED_ADJACENT_ATMOS)
```

Comment:
if you're gonna do this, change the use in firelock code

Reply:
Alright.